CMakeLists.txt

```diff
@@ -239,8 +239,8 @@ include_directories(SYSTEM
 
 add_subdirectory(src)
 
-if(OGRE_VERSION VERSION_LESS "1.10")
-  set(EXCLUDE_PATTERN "ogre1.10.*")
+if(OGRE_VERSION VERSION_LESS "1.12")
+  set(EXCLUDE_PATTERN "ogre1.12.*")
```

Comment:
The syntax of the font definition file was already changed in OGRE 1.10:
https://ogrecave.github.io/ogre/api/1.10/_font-_definition-_scripts.html

So why change 1.10 to 1.12 here?